test/relations.test.js

```diff
@@ -3790,7 +3790,7 @@ describe('relations', function() {
           return Person.findById(personId);
         })
         .then(function(person) {
-          person.passportItem().toObject().should.eql({name: 'Jim'});
+          person.passportItem().toObject().should.eql({name: 'Mitsos'});
```

Comment:
Yes, I think the name should be 'Jim'. 

My guess is that now with `updateOnLoad` is gone, `passport.save` updates the parent instance`person` in a different way, perhaps it does not update all properties as need. If I am correct, then this is a latent bug in the implementation of embedded relations that you have just discovered :)

You can also use `git blame` (or the github UI) to see who wrote this test and ask that person directly.